docs/source/Getting-Started/Running-Keystone-on-Hardware.rst

```diff
@@ -33,11 +33,11 @@ SiFive-compatible Linux and SM.
 This includes some patches for DTB compatibility.
 Also, the build will forcibly use initramfs for a simpler deployment.
 
-Once you have built the image, you will see ``riscv-pk.build/bbl`` under your
-build directory.
+Once you have built the image, you will see 
+``sm.build/platform/generic/firmware/fw_payload.bin`` under your build directory.
 
-Separately, ``make image`` will also generate ``bbl.bin`` under your build directory.
-This is the file that you want to reflash the SD card with.
+Separately, ``make image`` will also generate ``fw_payload.bin`` under your build
```

Comment:
This is not true, make image will not generate fw_payload.bin under your build directory.

Reply:
Ack. Removed mentioning of `make image` as it is not needed. I can see the fw_payload.bin after just a `make`.